src/HRQLS/Controllers/Food/Sanitation.php

```diff
@@ -22,12 +22,12 @@
      * @var array
      */
     public $cities = [
-        'norfolk' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=norfolk',
-        'portsmouth' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=portsmouth',
-        'virginiabeach' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=virginia%20beach',
-        'suffolk' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=suffolk',
-        'hampton' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=hampton',
-        'chesapeake' => 'https://ohi-api.code4hr.org/vendors?after=24-10-2014&city=chesapeake'
+        'norfolk' => 'https://ohi-api.code4hr.org/vendors?after=01-01-2016&city=norfolk',
```

Comment:
To reduce the maintenance needed to keep data current can we simply do this for the URL's
```php
$startDate = new DateTime('january first of last year')->format('n-j-Y');
'norfolk' => "https://ohi-api.code4hr.org/vendors?after={$startDate}&city=norfolk"
```